keywords.robot

```diff
@@ -343,13 +349,23 @@ Prepare Test Suite
     ...                SSH and serial connections, setting current platform to
     ...                global variable and setting Device Under Test to start
     ...                state. Keyword used in all [Suite Setup] sections.
-    Run Keyword If   '${config[:4]}' == 'apu2'    Import Resource    ${CURDIR}/platform-configs/apu2.robot
-    Run Keyword If   '${dev_type}' != 'auto'    Set Storage Device Number And Type
+    Run Keyword If   '${config[:4]}' == 'apu2'    Run Keywords    Import Resource
+    ...        ${CURDIR}/platform-configs/apu2.robot
+    ...        AND    Set Library Search Order    apu2
+    ...    ELSE IF   '${config[:6]}' == 'asrock'  Run Keywords    Import Resource
+    ...        ${CURDIR}/platform-configs/asrock-r1000v.robot
+    ...        AND    Set Library Search Order    asrock-r1000v
+    ...    ELSE IF    '${config[:10]}' == 'supermicro'    Run Keywords    Import Resource
+    ...        ${CURDIR}/platform-configs/supermicro-m11sdv-8ct-ln4f.robot
+    ...        AND    Set Library Search Order    supermicro-m11sdv-8ct-ln4f
+    Run Keyword If   '${dev_type}' not in ['auto', 'None']
+    ...               Set Storage Device Number And Type
 
     Open Connection And Log In
     ${platform}=    Get current RTE param    platform
     Set Global Variable    ${platform}
     Get DUT To Start State
+    Telnet.Set Timeout    90s
```

Comment:
@pkonkol It should be moved to test Settings, it is used in each test.

Reply:
Moved